app/scripts/views/base.js

```diff
@@ -92,23 +92,39 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
      */
     layoutClassName: null,
 
+    /**
+     * The default screen name
+     *
+     * @property screenName
+     */
+    screenName: '',
+
     constructor: function (options) {
       options = options || {};
 
-      this.subviews = [];
```

Comment:
I couldn't help but take the opportunity to alphabetize the list of items that don't depend on `this.window`.